packages/tasks/src/model-libraries.ts

```diff
@@ -150,6 +150,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"adapter_config.json"`,
 	},
+	comfy: {
+		prettyLabel: "ComfyUI",
+		repoName: "ComfyUI",
+		repoUrl: "https://github.com/comfyanonymous/ComfyUI",
+		filter: false,
+		countDownloads: `path_extension:"safetensors"`,
```

Comment:
This will increase the download counts a bit - but unsure what should be the best way to track downloads otherwise.

If we use `safetensors` then it'd be same as the way we track multiple GGUFs in a repo.

Reply:
makes sense